src/main/java/co/org/osso/api/CalculatorService.java

```diff
@@ -0,0 +1,60 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CalculatorService {
+
+
+    public List getFibonacci(Integer number) {
+
+        List<Integer> fibonacci = new ArrayList<>();
+        fibonacci.add(1);
+        fibonacci.add(1);
+
+        for ( int i = 2; i < number; i++ ){
+            fibonacci.add(i, fibonacci.get(i-1) + fibonacci.get(i-2));
+        }
+
+        return fibonacci;
+
+    }
+
+    public int getFactorial(int number) {
+        int factorial = 1;
+        for (int i = 1; i <= number; i++)
+            factorial = factorial * i;
+
+        return factorial;
+    }
+
+
+    public double getOperationResult(String operation, double n1, double n2) {
+
+        double result=0;
```

Comment:
We don't need this variable, nor do we need the `break;` below if we just `return` in the `switch` `case`s